crates/bitwarden-crypto/src/store/context.rs

```diff
@@ -127,29 +127,26 @@ impl<Ids: KeyIds> KeyStoreContext<'_, Ids> {
     // TODO: All these encrypt x key with x key look like they need to be made generic,
     // but I haven't found the best way to do that yet.
 
-    /// Decrypt a symmetric key into the context by using an already existing symmetric key
+    /// Unwrap a symmetric key into the context by using an already existing symmetric key
     ///
     /// # Arguments
     ///
-    /// * `encryption_key` - The key id used to decrypt the `encrypted_key`. It must already exist
-    ///   in the context
+    /// * `wrapping_key` - The key id used to unwrap the `key_to_unwrap`. It must already exist in
+    ///   the context
     /// * `new_key_id` - The key id where the decrypted key will be stored. If it already exists, it
     ///   will be overwritten
-    /// * `encrypted_key` - The key to decrypt
+    /// * `key_to_unwrap` - The key to decrypt
```

Comment:
There's still a reference to decryption instead of unwrapping (`The key to decrypt` -> `The key to unwrap`). To avoid the repetition with the comment, we could also rename the parameter to more closely resemble it's type too:
```suggestion
    /// * `wrapped_key` - The key to unwrap
```